docs/content/anchoredPosition.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Anchored Position Behavior
+---
+
+The `getAnchoredPosition` behavior and `useAnchoredPosition` hook are used to calculate the position of a "floating" element that is anchored to another DOM element. This is useful for implementing overlay UI, such as dialogs, popovers, tooltips, toasts, and dropdown-style menus.
+
+At a high level, the `getAnchoredPosition` routine will attempt to find the most suitable position for the floating element based on the passed-in settings, its containing element, and the size and position of the anchor element. Specifically, the calculated position should try to ensure that the floating element, when positioned at the calculated coordinates, does not overflow or underflow the container's bounding box.
```

Comment:
I'm not familiar with the word `routine` in the context of programming, is it a generic word for behavior/function?

Reply:
Yes, synonyms. I think I probably used the word `routine` to signify that I'm talking about the high-level algorithm rather than the specific implementation, which might be implied by the word `function`. I can change this to `algorithm`.